app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CompaniesController < ApplicationController
+  def new
+    @company = Company.new
+  end
+
+  def create
+    @company = Company.new(company_params)
+    @company.client = current_client
```

Comment:
Como o client has one company daria pra fazer algo assim, o que acha?

```suggestion
    @company = current_client.build_company(company_params)
```

Reply:
Inserimos a sugestão e passou, obrigado!